venter/admin.py

```diff
@@ -96,41 +97,32 @@ def send_emails(modeladmin, request, queryset):  # pylint: disable=R0201
         mailing_list = []
 
         for item in queryset:
-            # Check if the complaint has a valid authority set
-            if item.authority_email is None:
```

Comment:
You need to check if no authorities are selected here instead and skip the whole thing if this is the case.

Reply:
This has been done.